Package.swift

```diff
@@ -39,3 +41,16 @@ extension Package.Dependency {
 extension Target.Dependency {
     static let dependencies: Self = .product(name: "Dependencies", package: "swift-dependencies")
 }
+
+extension [SwiftSetting] {
```

Comment:
There are a few more upcoming flags that aren't in this list. Did we want the full list here if this setting is going to mimic Swift 6.0? The 2 specifically I was looking at were:

https://github.com/apple/swift-evolution/blob/main/proposals/0413-typed-throws.md
https://github.com/apple/swift-evolution/blob/main/proposals/0418-inferring-sendable-for-methods.md

Reply:
Those would require another increase in the minimum Swift version and may only be available in 5.10+. I don't think either of them would make much difference for this package anyway.